calendar-tool/tool/src/java/org/sakaiproject/calendar/entityproviders/CalendarEventEntityProvider.java

```diff
@@ -219,4 +229,32 @@ public CalendarEventDetails getCalendarEventDetails(EntityView view) {
 		}
 	}
 
+	/**
+	 * Returns TimeRange for an year.
```

Comment:
This should be in the `@return` and the description should  be of what this is trying to achieve, eg. Builds a timerange that goes from